db/seedfiles/notice_triggers_seed.rb

```diff
@@ -468,320 +425,407 @@
-        notice_template: 'notices/ivl/documents_verification_reminder1',
-        notice_builder: 'IvlNotices::ConsumerNotice',
-        mpi_indicator: 'MPI_IVLV20A',
-        notice_trigger_element_group: {
-          market_places: ['individual'],
-          primary_recipients: ["consumer"],
-          primary_recipient_delivery_method: ["secure_message", "paper"],
-          secondary_recipients: []
-        }
-      }
-    ]
-  },
-  {
-    hbx_id: 'Notice20B',
-    title: 'Request for Additional Information - Second Reminder',
-    description: 'After 25 days passed, notice to be sent to Consumers informing them of the outstanding verifications',
-    resource_name: 'consumer_role',
-    event_name: 'second_verifications_reminder',
-    notice_triggers: [
-      {
-        name: 'Second Outstanding Verification Notification',
-        notice_template: 'notices/ivl/documents_verification_reminder2',
-        notice_builder: 'IvlNotices::ConsumerNotice',
-        mpi_indicator: 'MPI_IVLV20B',
-        notice_trigger_element_group: {
-          market_places: ['individual'],
-          primary_recipients: ["consumer"],
-          primary_recipient_delivery_method: ["secure_message", "paper"],
-          secondary_recipients: []
-        }
-      }
-    ]
-  },
-  {
-    hbx_id: 'Notice21',
-    title: 'Request for Additional Information - Third Reminder',
-    description: 'After 50 days passed, notice to be sent to Consumers informing them of the outstanding verifications',
-    resource_name: 'consumer_role',
-    event_name: 'third_verifications_reminder',
-    notice_triggers: [
-      {
-        name: 'Third Outstanding Verification Notification',
-        notice_template: 'notices/ivl/documents_verification_reminder3',
-        notice_builder: 'IvlNotices::ConsumerNotice',
-        mpi_indicator: 'MPI_IVLV21',
-        notice_trigger_element_group: {
-          market_places: ['individual'],
-          primary_recipients: ["consumer"],
-          primary_recipient_delivery_method: ["secure_message", "paper"],
-          secondary_recipients: []
-        }
-      }
-    ]
-  },
-  {
-    hbx_id: 'Notice22',
-    title: 'Request for Additional Information - Fourth Reminder',
-    description: 'After 65 days passed, notice to be sent to Consumers informing them of the outstanding verifications',
-    resource_name: 'consumer_role',
-    event_name: 'fourth_verifications_reminder',
-    notice_triggers: [
-      {
-        name: 'Fourth Outstanding Verification Notification',
-        notice_template: 'notices/ivl/documents_verification_reminder4',
-        notice_builder: 'IvlNotices::ConsumerNotice',
-        mpi_indicator: 'MPI_IVLV22',
-        notice_trigger_element_group: {
-          market_places: ['individual'],
-          primary_recipients: ["consumer"],
-          primary_recipient_delivery_method: ["secure_message", "paper"],
-          secondary_recipients: []
-        }
-      }
-    ]
-  },
+# ivl_notice_triggers = [
```

Comment:
we do not necessary to comment it out 
since it is in IVL. anyway we do not triggering right